src/service/upstart/deb/brooklyn.conf

```diff
@@ -0,0 +1,45 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
@alrick87 use `git mv deb-packaging/src/conf/brooklyn.conf src/service/upstart/deb/brooklyn.conf` to preserve history.